src/index.js

```diff
@@ -178,7 +179,8 @@ function configureToMatchImageSnapshot({
     const snapshotsDir = customSnapshotsDir || path.join(path.dirname(testPath), SNAPSHOTS_DIR);
     const diffDir = customDiffDir || path.join(snapshotsDir, '__diff_output__');
     const baselineSnapshotPath = path.join(snapshotsDir, `${snapshotIdentifier}-snap.png`);
-
+    ObsoleteReporter.markTouchedFile(baselineSnapshotPath);
```

Comment:
Can we add documentation about this in the readme?

Would be nice to have an integration test but can see why it would be challenging to do.

Reply:
Sure 👍